dashboard/app/controllers/code_reviews_controller.rb

```diff
@@ -59,4 +59,40 @@ def update
 
     render json: code_review.summarize_with_comments
   end
+
+  # GET /code_reviews/from_peers
+  # Returns the list of open code reviews for the given script and level
```

Comment:
This description doesn't seem quite right to me, because the endpoint is returning peer information not code review information. Saying "Returns a list of peers with open code reviews for the given script and level" seems more correct to me

Reply:
Yeah, maybe the url path is wrong

Reply:
As we discussed offline, changing the url to `/code_reviews/peers_with_open_reviews` to clarify that we are returning information about the peer.